internal/store/api_keys.go

```diff
@@ -42,3 +46,17 @@ func (s *Store) APIKeys(orgID string, opts *atlas.ListOptions) ([]atlas.APIKey,
 		return nil, fmt.Errorf("unsupported service: %s", s.service)
 	}
 }
+
+// APIKey encapsulate the logic to manage different cloud providers
+func (s *Store) APIKey(orgID, apiKeyID string) (*atlas.APIKey, error) {
```

Comment:
I'll probably add the organization project keys here too -- are you ok if I change these in my pr to prefix with `Project`?